### ISSUE TITLE
Unable to compile code due to SDL2 problem

### ISSUE THREAD

**Issue by MicheleIsoli** [nrootconauto/Aiwnios#15]
Hi, just got an error while compiling the code.
I don't know why tho, here's the error:

```
[100%] Built target aiwnios
make[2]: *** [CMakeFiles/HCRT2.BIN.dir/build.make:71: CMakeFiles/HCRT2.BIN] Errore 1
make[1]: *** [CMakeFiles/Makefile2:204: CMakeFiles/HCRT2.BIN.dir/all] Errore 2
make: *** [Makefile:156: all] Errore 2
```

Also, a messagebox spawn saying "Unable to init SDL".
![immagine](https://github.com/user-attachments/assets/ae1094ad-16cb-4da6-a6fc-9ff1a54ae5e9)

Please, can you help me fix it?

**Comment by austings** [comment]
do you have sdl lib 2

**Comment by MicheleIsoli** [comment]
Yes i installed it using this istructions: [https://wiki.libsdl.org/SDL2/Installation](https://wiki.libsdl.org/SDL2/Installation)

**Comment by nrootconauto** [comment]
Ill look,tell me what OS/distro you are using during the meantime

**Comment by MicheleIsoli** [comment]
Kali Linux 2024.4 which is basically Debian 12

**Comment by nrootconauto** [comment]
I read the message,the issue seems to be the HCRT2.BIN (the runtime) is not being built. Are you are ARM64/X86_64. Try running `./aiwnios -b`(bootstrap the runtime binary) in the root directory and show me the output. Its probably an issue with the CMakeLists.txt being in the wrong directory.

![image](https://github.com/user-attachments/assets/19f4d29a-14a6-42b7-b128-7ed6941ed914)

Ill try to resolve it

**Comment by nrootconauto** [comment]
I would hate to be a pain,but can you do `gdb --args ./aiwnios -db ` ,press `r` then press `bt` once it faults(?).
It should look like this(I purposely made it fail for an example)
![image](https://github.com/user-attachments/assets/da819696-0cac-44f5-9711-577b823f834f)

**Comment by nrootconauto** [comment]
Ok,I made the message box more useful. I also initialize the thing with only the useful subsytem,can you try the latest commit (https://github.com/nrootconauto/Aiwnios/commit/14276359add88b73aec481bb2d7cc7a65f205cb1) @MicheleIsoli

**Comment by sieveprime** [comment]
@MicheleIsoli checking in to see if you were you able to build with the latest commit

**Comment by nrootconauto** [comment]
Dead issue?
